lib/src/form_builder_validators.dart

```diff
@@ -201,25 +201,28 @@ class FormBuilderValidators {
     };
   }
 
+  // TODO(any): l10n
   /// [FormFieldValidator] that requires the field's value to be a valid integer.
-  static FormFieldValidator integer({
+  static FormFieldValidator integer(
+    BuildContext context, {
     String errorText = 'Value must be an integer.',
   }) {
     return (valueCandidate) {
-      if (int.tryParse(valueCandidate) == null && valueCandidate.isNotEmpty) {
+      if (int.tryParse(valueCandidate ?? '') == null) {
         return errorText;
       }
       return null;
     };
   }
 
+  // TODO(any): l10n
   /// [FormFieldValidator] that requires the field's value to be a valid double.
-  static FormFieldValidator double_({
+  static FormFieldValidator double_(
```

Comment:
How about renaming this to be `isDouble` or `validDouble` to avoid the conflict with the type name?

Reply:
Yes, why not, but that would make the name inconsistent with others.

Reply:
How is this really different from the old `numeric` validator?

Reply:
It should also be renamed integer to isInteger and numeric to isNumeric to keep consistency.

But it is true that this would make things clearer, it is a validator for a valid but not REQUIRED number/integer/double.

Reply:
Do we really need to have three digital validators? Or is double and integer sufficient?

Reply:
I'm happy to defer the _naming decision_ until @danvick provides feedback.  I just really don't like identifiers other than `_` ending in an underscore.